tests/com/twitter/intellij/pants/integration/OSSProjectInfoResolveTest.java

```diff
@@ -78,13 +80,14 @@ public void testTargetJars() {
   }
 
   public void testListTargets() {
-    assertEquals(
-      Arrays.asList(
-        "examples/src/java/org/pantsbuild/example/hello/main:main",
-        "examples/src/java/org/pantsbuild/example/hello/main:readme",
-        "examples/src/java/org/pantsbuild/example/hello/main:main-bin"
-      ),
+    HashSet<String> targetsInBuild = Sets.newHashSet(
       PantsUtil.listAllTargets(myProjectRoot.getPath() + File.separator + "examples/src/java/org/pantsbuild/example/hello/main/BUILD")
     );
+    assertEquals(
+      Sets.newHashSet("examples/src/java/org/pantsbuild/example/hello/main:main",
+                              "examples/src/java/org/pantsbuild/example/hello/main:readme",
+                              "examples/src/java/org/pantsbuild/example/hello/main:main-bin"),
```

Comment:
Should these two lines be indented less?

Reply:
ha good catch! my auto format shortcut isn't working :D